code/modules/reagents/chemistry/recipes.dm

```diff
@@ -339,8 +397,10 @@
 		if((reagent.type in required_reagents) || (reagent.type in results))
 			reagents.add_reagent(reagent.type, reagent.volume, added_purity = reagent.purity, no_react = TRUE)
 			holder.remove_reagent(reagent.type, reagent.volume)
+	if(!force_range)
+		force_range = (sum_volume/5) + 3
 	if(reagents.reagent_list)
-		smoke.set_up(reagents, (sum_volume/5), holder.my_atom)
+		smoke.set_up(reagents, , holder.my_atom)
```

Comment:
is the middle parameter here supposed to be blank?

Reply:
oops! Thanks for the catch!